R/classes.R

```diff
@@ -284,6 +284,47 @@ private=list(
     }
 ))
 
+AzureTokenWorkload <- R6::R6Class("AzureTokenWorkload", inherit=AzureToken,
```

Comment:
might wanna add docs on the class and methods?

Reply:
It inherits the super docs. Do you think additional ones are needed?